### DISCUSSION TITLE
System messages - allow switching language of messages from list

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#8966] [Ideas]
Right now we can only switch message language in message edit modal. It would be useful if we could switch also while browsing messages list.

**Comment by schwarzdesign** [comment]
@piotrpog Just to be sure, do you mean the edit modals under Utilities -> System Messages? Because I don't see a way to change the language for the message at all. Instead, the messages are always in the _preferred language of my user account_. Maybe it's because I only have one site defined?

My setup: I have one site defined with its language set to German. My admin account is set to English. When I go to Utilities -> System Messages I can only edit the messages in English. If I want to change the System Messages in German, I have to switch my account preference to German. @brandonkelly Is this intended behaviour or a bug?

**Comment by brandonkelly** [comment]
@schwarzdesign That’s a bug. Just posted as a new issue (#7992) and it’s now fixed for the next release.
